tools/eosshellcontent.py

```diff
@@ -51,6 +52,8 @@ class ShellContent:
 
     def __init__(self):
         self._translations, self._langs = self._get_translations_dict()
+        # Regex for matching anything starting with a <tag> like format
+        self._tag_expression = re.compile('^\s*\<\w+\>.*')
```

Comment:
AFAIU it could start with a <ul> too (possibly others)... I didn't find anything specific in the spec so I chose to be a bit more flexible and allow anything that looks like a < tag > .